City-of-Helsinki/helsinki-paatokset
# PP-0: Drupal core and other package updates

**To test:**
* Checkout branch
* Run `make fresh` (if you have an existing database dump)
* Make sure all modules and Drupal core is actually up-to-date: https://helsinki-paatokset.docker.so/fi/admin/reports/updates
* Test that the site works and nothing weird was introduced in the platform config updates.

**List of updates:**
```
- Upgrading composer/composer (2.1.3 => 2.1.5)
- Upgrading composer/xdebug-handler (2.0.1 => 2.0.2)
- Upgrading doctrine/collections (1.6.7 => 1.6.8)
- Upgrading drupal/core (9.2.0 => 9.2.4)
- Upgrading drupal/core-composer-scaffold (9.2.0 => 9.2.4)
- Upgrading drupal/core-dev (9.2.0 => 9.2.4)
- Upgrading drupal/core-recommended (9.2.0 => 9.2.4)
- Upgrading drupal/easy_breadcrumb (1.15.0 => 2.0.1)
- Upgrading drupal/entity_usage (2.0.0-beta3 => 2.0.0-beta4)
- Upgrading drupal/field_group (3.1.0 => 3.2.0)
- Upgrading drupal/gin (3.0.0-alpha35 => 3.0.0-alpha36)
- Upgrading drupal/gin_toolbar (1.0.0-beta17 => 1.0.0-beta19)
- Upgrading drupal/hdbt (1.3.9 => 1.3.11)
- Upgrading drupal/hdbt_admin (1.3.4 => 1.3.7)
- Downgrading drupal/helfi_media_map (1.x-dev ccf18c8 => 1.0.0)
- Upgrading drupal/helfi_platform_config (1.2.12 => 1.3.1)
- Upgrading drupal/helfi_tpr (1.3.3 => 1.3.4)
- Upgrading drupal/scheduler (1.3.0 => 1.4.0)
- Upgrading drupal/search_api (1.19.0 => 1.20.0)
- Upgrading drupal/twig_tweak (3.1.1 => 3.1.2)
- Upgrading drush/drush (10.5.0 => 10.6.0)
- Upgrading elasticsearch/elasticsearch (v7.13.1 => v7.14.0)
- Upgrading justinrainbow/json-schema (5.2.10 => 5.2.11)
- Upgrading mikey179/vfsstream (v1.6.8 => v1.6.9)
- Upgrading nikic/php-parser (v4.10.5 => v4.12.0)
- Upgrading pear/archive_tar (1.4.13 => 1.4.14)
- Upgrading phar-io/manifest (2.0.1 => 2.0.3)
- Upgrading ruflin/elastica (dev-master cbc962e => dev-master e137d9f)
- Upgrading sirbrillig/phpcs-variable-analysis (v2.11.1 => v2.11.2)
- Upgrading symfony/browser-kit (v4.4.25 => v4.4.27)
- Upgrading symfony/css-selector (v4.4.25 => v4.4.27)
- Upgrading symfony/dom-crawler (v4.4.25 => v4.4.27)
- Upgrading symfony/filesystem (v4.4.25 => v4.4.27)
- Upgrading symfony/finder (v4.4.25 => v4.4.27)
- Upgrading symfony/lock (v4.4.25 => v4.4.27)
- Upgrading symfony/phpunit-bridge (v5.3.0 => v5.3.4)
- Upgrading theseer/tokenizer (1.2.0 => 1.2.1)
```
